templates/docker-compose.yml

```diff
@@ -72,7 +75,6 @@ services:
     # entrypoint: /sbin/tini -- /usr/local/bin/pict-rs -p /mnt -m 4 --image-format webp
     networks:
       - lemmyinternal
-      - lemmyexternalproxy
```

Comment:
pictrs needs to be able to connect outbound. It caches external images. If removing access to this network removes outbound connectivity, then this would break pictrs fetching images.

Reply:
Gotcha, I had originally started making this commit during the 0.18 rollout when some things were broken immediately after. I'll amend the PR to reflect the desired state.